controllers/authenticationController.js

```diff
@@ -0,0 +1,56 @@
+const {
+  signInView, validateAuthTokenView, hasSocialView,
+} = require('views/authenticationController');
+const { UserModel } = require('models');
+const render = require('concerns/render');
+const Strategies = require('utilities/operations/auth/authStrategies');
```

Comment:
create index file at oparation folder, and use destructurisation(like at waivio-api), if you dont understand, ask me, i will explain to you